apps/agent/src/mastra/workflows/github-notify-reviewers.ts

```diff
@@ -0,0 +1,136 @@
+import { Step, Workflow } from '@mastra/core/workflows'
+import { z } from 'zod'
+import { discordClient } from '../../lib/discord'
+import { GITHUB_REPO, githubClient } from '../../lib/github'
+import { PullRequest } from '../../lib/type'
+import { formatDate } from '../../utils/datetime'
+import { DISCORD_GITHUB_MAP } from '../../constants/discord'
+
+class NotifyReviewersWorkflow {
+  private workflow: Workflow
+
+  constructor() {
+    this.workflow = new Workflow({
+      name: 'Notify Reviewers',
+    })
+  }
+
+  private getPendingReviews = new Step({
+    id: 'get-pending-reviews',
+    execute: async () => {
+      const prs = await githubClient.getRepoPRs(GITHUB_REPO, {
+        from: formatDate(new Date()),
+        isMerged: false,
+        isOpen: true,
+      })
+
+      // get PRs with reviewer assigned but not reviewed yet
+      const todayPRsWithReviewerAssigned = prs.filter((pr) => {
+        return pr.requested_reviewers && pr.review_comments === 0
```

Comment:
Does it cover the case when a PR without comments but having approvals? in this case, it shouldn't send a notification

Also, looks like this check is only valid for a fresh PR. I think it should be more robust to cover case when a reviewer is requested. please taking the scenario:

1. A request B to review
2. B reviews the PR and makes comments to request changes
3. A submits an update and re-request B to review
4. if there is no actions (comments, approvals) since then, send a notification